### PR TITLE
CIE-4177. Added topic for security hardening.

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/edge/installation-bundle/edge-security-hardening.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+weight: 26
+title: Security harden the Edge appliance
+layout: redirect
+---
+The Edge appliance is security hardened, by default. Furthermore, there are certain security configuration that you can apply to enhance the security of the Edge appliance. To apply the security configuration, see [Security configuration](/edge/configuration/#configuring-security).
```
Comment (rssag) [inline content/edge/installation-bundle/edge-security-hardening.md:6]:
```suggestion

The Edge appliance is security hardened by default.
Furthermore, there are certain security configuration that you can apply to enhance the security of the Edge appliance. To apply the security configuration, see [Security configuration](/edge/configuration/#configuring-security). 
```

**Review by rssag — APPROVED** [review]

**Review by rssag — APPROVED** [review]

File: content/edge/installation-bundle/edge-security-hardening.md [hunk 2]
```diff
@@ -5,5 +5,4 @@ layout: redirect
 ---
 
 The Edge appliance is security hardened by default.
-Furthermore, there are certain security configuration that you can apply to enhance the security of the Edge appliance. To apply the security configuration, see [Security configuration](/edge/configuration/#configuring-security). 
-For information about security configuration guidelines, see [Security Hardening Guidelines](/files/edge/10-13-0_Security_Hardening_Guidelines_guide.pdf).
+Furthermore, there are certain security configurations that you can apply to enhance the security of the Edge appliance. To apply the security configuration, see [Security configuration](/edge/configuration/#configuring-security).
```
[the remark this answers is not in the document]
Comment (sharath-jk) [reply content/edge/installation-bundle/edge-security-hardening.md:8]: Done.

**Review by bbyreddy — APPROVED** [review]

File: content/edge/configuration-bundle/configuring-security.md [hunk 3]
```diff
@@ -25,12 +30,12 @@ The Edge appliance is security hardened, by default. Furthermore, there are cert
     |<div style="width:100px">Configuration</div>|Category|Parameter|Description
     |:---|:---|:---|:---
     |Operating system|SELinux|**SELinux mode**|SELinux secures the operating system from unprivileged actions of confined users. When enabled, SELinux has two modes: permissive and enforcing.<br><br>**permissive**: In the permissive mode, SELinux policy is not enforced. The system remains operational and SELinux does not deny any operations but only logs AVC messages, which can be then used for troubleshooting, debugging, and SELinux policy improvements. Each AVC is logged only once in this case.<br><br>**enforcing**: In the enforcing mode, it enforces the SELinux policy and denies access based on SELinux policy rules.<br><br>{{< c8y-admon-important >}}
-  In the enforcing mode, you cannot access the Edge appliance remotely through VNC.<br>{{< /c8y-admon-important >}}<br>Default value: permissive
+    In the enforcing mode, you cannot access the Edge appliance remotely through VNC.<br>{{< /c8y-admon-important >}}<br>Default value: permissive
```
Comment (rssag) [inline content/edge/configuration-bundle/configuring-security.md:33]:
```suggestion
In the enforcing mode, you cannot access the Edge appliance remotely through VNC.
    {{< /c8y-admon-important >}}<br>Default value: permissive
```
indentation inside admonition blocks is rendered, unfortunately.

File: content/edge/configuration-bundle/configuring-security.md [hunk 4]
```diff
@@ -25,12 +30,12 @@ The Edge appliance is security hardened, by default. Furthermore, there are cert
...
     ||SSH|**SSH enabled**|Enables or disables the SSH functionality of the Edge appliance. By default, the SSH functionality is enabled.<br><br>Changing the value to false disables the SSH functionality. When disabled, you will not be able to SSH into the Edge appliance, locally or remotely.<br><br>Default value: True
     ||Session inactivity|**Login sessions inactivity timeout (seconds)**|The idle duration before a user session is terminated. When configured, the user session terminates after the specified idle duration.<br><br>Default value: 600 seconds<br>Minimum value: 0<br>
     ||Audit logging|**Audit logging enabled**|Enables or disables audit logging. By default, audit logging is disabled. A value of true enables audit logging. Enabling audit logging increases the consumption of disk space.<br><br>{{< c8y-admon-important >}}
-  Once enabled, you cannot disable the audit logging configuration.
-  {{< /c8y-admon-important >}}<br>Default value: False
+    Once enabled, you cannot disable the audit logging configuration.
```
Comment (rssag) [inline content/edge/configuration-bundle/configuring-security.md:37]:
```suggestion
Once enabled, you cannot disable the audit logging configuration.
```

**Comment by sharath-jk** [comment]
@bbyreddy - I'll keep this PR open until we have the security hardening guide updated by Saran.

**Review by rssag — CHANGES_REQUESTED** [review]

File: content/edge/configuration-bundle/configuring-security.md [hunk 5]
```diff
@@ -1,10 +1,15 @@
...
+weight: 10
+title: Configuring security
 layout: redirect
 ---
 
-The Edge appliance is security hardened, by default. Furthermore, there are certain security configuration that you can apply to enhance the security of the Edge appliance. To apply the security configuration, follow the steps below: 
+The Edge appliance is security hardened by default.
+Furthermore, there are certain security configurations that you can apply to enhance the security of the Edge appliance.
+
+{{< c8y-admon-important >}}For information about Deployment seurity best practices and Tenant administration security configuration guidelines, see <a href="/files/edge/10-13-0_Security_Hardening_Guidelines_guide.pdf" target="_blank">Security Hardening Guidelines</a>.{{< /c8y-admon-important >}}
```
Comment (rssag) [inline content/edge/configuration-bundle/configuring-security.md:10]:
```suggestion
{{< c8y-admon-important >}}
For information about deployment security best practices and tenant administration security configuration guidelines, see <a href="/files/edge/10-13-0_Security_Hardening_Guidelines_guide.pdf" target="_blank">Security Hardening Guidelines</a>.
{{< /c8y-admon-important >}}
```

**Review by rssag — APPROVED** [review]

File: content/edge/configuration-bundle/configuring-security.md [hunk 6]
```diff
@@ -1,10 +1,17 @@
...
-weight: 86
-title: Security configuration
+weight: 10
+title: Configuring security
 layout: redirect
 ---
 
-The Edge appliance is security hardened, by default. Furthermore, there are certain security configuration that you can apply to enhance the security of the Edge appliance. To apply the security configuration, follow the steps below: 
+The Edge appliance is security hardened by default.
+Furthermore, there are certain security configurations that you can apply to enhance the security of the Edge appliance.
```
Comment (MWindel) [inline content/edge/configuration-bundle/configuring-security.md:8]:
```suggestion
Furthermore, you can apply specific security configurations to enhance the security of you Edge appliance.
```

**Pull request merged**